uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1020.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1020 {
+
+  /**
+   * Rename Line Manager Relationship
+   */
+  public function upgrade_1020() {
+    $this->up1020_renameLineManagerRelationship();
+
+    return TRUE;
+  }
+
+  /**
+   * Renaming Line Manager Relationship
+   */
+  private function up1020_renameLineManagerRelationship() {
+    $relationshipType = civicrm_api3('RelationshipType', 'get', [
+      'name_b_a' => 'Line Manager',
+    ]);
+    if (empty($relationshipType['values'])) {
```

Comment:
This works fine, but since you're using `$relationshipType['id']` in the next line why not check for this here instead, just makes more sense when you're reading it